README.md

```diff
@@ -44,31 +37,34 @@ An Office add-in includes two basic components: an XML manifest file, and your o
 
 When a add-in project is created, you can explore and customize the components by reviewing the key files listed below.
 
-- The `./manifest.xml` file in the root directory of the project defines the settings and capabilities of the add-in.  <br>You can check whether your manifest file is valid by selecting `Validate Manifest File` in the `Office Add-ins Development Kit` extension tree view.
+- The `./manifest.xml` file in the root directory of the project defines the settings and capabilities of the add-in.  <br>You can check whether your manifest file is valid by selecting **Validate Manifest File** option from the Office Add-ins Development Kit.
 - The `./src/taskpane/taskpane.html` file contains the HTML markup for the task pane.
 - The `./src/taskpane/taskpane.css` file contains the CSS that's applied to content in the task pane.
-- The `./src/taskpane/taskpane.js` file contains the Office JavaScript API code that facilitates interaction between the task pane and the Word application.
+- The `./src/taskpane/taskpane.js` file contains the Office JavaScript API code that facilitates interaction between the task pane and the Office application.
 
 ### Preview your add-in in Office apps
 
 To understand how the add-in will work in Office apps, you can use the Office Add-ins Development Kit to easily run and debug your Office add-in in your local dev environment.
-<br><img src=./assets/Dev_Kit_Preview.gif>
-
+<br><img src=./assets/Dev_Kit_Preview.gif alt="Preview an add-in using Office Add-ins Development Kit"/>
 
 #### Preview Your Office Add-in (F5)
 
-Select `Preview Your Office Add-in(F5)` on the side panel to start running and debugging the add-in code. The Office Add-ins Development Kit will firstly chenck and install dependencies, and launch A Word/Excel/PowerPoint app to sideload the add-in sample.
-- You can also start debugging by hitting the `F5` key or running `npm run start` command in the terminal.
-- To debug on Office on the web, go to [Sideload Office Add-ins to Office on the web](https://learn.microsoft.com/office/dev/add-ins/testing/sideload-office-add-ins-for-testing)
-- To debug in Desktop (Edge Legacy), go to [Debug Edge Legacy Webview](https://learn.microsoft.com/office/dev/add-ins/testing/debug-add-ins-using-devtools-edge-legacy)
+Select **Preview Your Office Add-in(F5)** to launch the add-in and debug the code. In the drop down menu, select the option **Desktop (Edge Chromium)**.
 
-#### Stop Previewing Your Office Add-in
+The extension then checks that the prerequisites are met before debugging starts. Check the terminal for detailed information if there are issues with your environment. After this process, the Office desktop application launches and sideloads the add-in.
+
+You can also start debugging by hitting the `F5` key or running `npm run start` command in the terminal.

```

Comment:
Avoid the word "hitting" in favor of "pressing". It will translate better.
```suggestion
You can also start debugging by pressing the `F5` key or running the `npm run start` command in the terminal.
```